src/lambda/commands/createNewSamApp.ts

```diff
@@ -352,21 +352,26 @@ async function validateSamCli(samCliValidator: SamCliValidator): Promise<void> {
 
 export async function getProjectUri(
     config: Pick<CreateNewSamAppWizardResponse, 'location' | 'name'>,
-    file: string
+    files: string[]
 ): Promise<vscode.Uri | undefined> {
-    const cfnTemplatePath = path.resolve(config.location.fsPath, config.name, file)
-    if (await fileExists(cfnTemplatePath)) {
-        return vscode.Uri.file(cfnTemplatePath)
-    } else {
-        vscode.window.showWarningMessage(
-            localize(
-                'AWS.samcli.initWizard.source.error.notFound',
-                'Project created successfully, but {0} file not found: {1}',
-                file,
-                cfnTemplatePath
-            )
-        )
+    let file: string = files[0]
+    let cfnTemplatePath: string = path.resolve(config.location.fsPath, config.name, file)
+    for (let i = 0; i < files.length; i++) {
```

Comment:
doesn't look like `i` is being used for anything except iteration, so this could be:
```suggestion
    for (const file of files) {
```

Reply:
Refactored.